### ISSUE TITLE
Questions about the Implementation Details of AVID

### ISSUE THREAD

**Issue by AlonzoLeeeooo** [zhang-zx/AVID#4]
Hello,

Thanks for your amazing work! I have some questions about the implementation details of AVID. It would be highly appreciated if you could provide the answers. Here are my questions:

(1) The first stage training of AVID aims to update the motion module, similar to AnimateDiff. I notice that your model has additional inputs, i.e., masked latents and masks. Therefore, the input channels of U-net model will change from 4 (only latents) to 9 (latents, masked latents, and masks), where the pre-trained T2I model weights are thus not strictly the same. I wonder what strategies do you use to solve this problem?

(2) The second stage training of AVID aims to incorporate the structure guidance, similar to ControlNet. I notice that the input of the structure information extractor becomes the concatenation of latents, masked latents, masks, and HED edges, which should be 10 channels in total. Compared to inputting a single HED edge, what would the functionality of this strategy?

(3) For each stage, what amount of GPU resources do you use to run the training code?

Thank you so much for your time and attention from your busy schedule. Hope everything goes well with your research!

Best regards,

**Comment by zhang-zx** [comment]
Thanks for your interest in our work.

For your first question, we used stable diffusion inpainting as our image backbone. Thus, the number of the input channels is originally 9.

We mostly follow the design choice of ControlNet for this part. 

If I recalled correctly, we use 8 GPUs for most of our experiments.
